frontend/src/bundles/common/components/user-photo-uploader/user-photo-cropper.tsx

```diff
@@ -0,0 +1,122 @@
+import 'cropperjs/dist/cropper.css';
+
+import React, { useCallback,useRef,useState } from 'react';
+import { type ReactCropperElement } from 'react-cropper';
+import { Cropper } from 'react-cropper';
+import { IconContext } from 'react-icons';
+import { BsSave } from 'react-icons/bs';
+import { FaFileUpload } from 'react-icons/fa';
+import { FaArrowRotateRight } from 'react-icons/fa6';
+import { GrZoomIn , GrZoomOut } from 'react-icons/gr';
+
+import styles from './styles.module.scss';
+
+interface UploadCropperProperties {
+    image: string | ArrayBuffer | null;
```

Comment:
In place where you use this component, you are checking if image exist, so I suggest null can be removed as possible type here